pkg/controllers/dynakube/otelc/statefulset/container.go

```diff
@@ -29,14 +29,21 @@ func getContainer(dk *dynakube.DynaKube) corev1.Container {
 		imageTag = defaultImageTag
 	}
 
+	var arg string
+	if dk.IsExtensionsEnabled() {
+		arg = fmt.Sprintf("--config=eec://%s:%d/otcconfig/prometheusMetrics#refresh-interval=5s&auth-file=%s", dk.ExtensionsServiceNameFQDN(), consts.OtelCollectorComPort, otelcSecretTokenFilePath)
+	} else {
+		arg = "--config=file:///osconfig/config.yaml"
+	}
+
 	return corev1.Container{
 		Name:            containerName,
 		Image:           imageRepo + ":" + imageTag,
 		ImagePullPolicy: corev1.PullAlways,
 		SecurityContext: buildSecurityContext(),
 		Env:             getEnvs(dk),
 		Resources:       dk.Spec.Templates.OpenTelemetryCollector.Resources,
-		Args:            []string{fmt.Sprintf("--config=eec://%s:%d/otcconfig/prometheusMetrics#refresh-interval=5s&auth-file=%s", dk.ExtensionsServiceNameFQDN(), consts.OtelCollectorComPort, otelcSecretTokenFilePath)},
+		Args:            []string{arg},
```

Comment:
if I'm not mistaken we need to append not replace, so it's 1 or another or both.